css/style.css

```diff
@@ -14,7 +14,7 @@ body {
 
 body {
   color: hsl(0, 0%, 100%);
-  background-color: white;
+  background-color: darkseagreen;
```

Comment:
Since you are updating the color, wouldn't it be good if you take the colors out to the root and define the colors for the page there itself?

Something of this sort 
```
:root{
    --primary-color: "primary-color"
    --secondary-color: "secondary-color"
// similar kind of thing
```
[This](https://www.w3schools.com/css/css3_variables.asp) can be useful to take a quick glance.

So by this we can refactor the color scheme to the root and becomes easily accessible across the code. 
Btw please confirm that on changing this style, other pages also reflect the similar styling to avoid inconsistency.

Thank you.

Reply:
@sohamsshah  It would be a good idea to take the color attribute out of the root but it is one of the way to style our page . 
The above code would render the home page so Changes made in background color would reflect on the home page only !

You are Welcome to ask more questions :)